src/effects.js

```diff
@@ -27,19 +27,29 @@ var
 	rfxtypes = /^(?:toggle|show|hide)$/,
 	rrun = /queueHooks$/;
 
-function raf() {
+function raf( timestamp ) {
 	if ( timerId ) {
 		window.requestAnimationFrame( raf );
-		jQuery.fx.tick();
+		jQuery.fx.tick( timestamp );
 	}
 }
 
+// We need to be using jQuery.now() or performance.now() consistently as they return different
+// values: performance.now() counter starts on page load.
+// Support: IE <10, Safari <8.0, iOS <9, Android <4.4, Node with jsdom 9.4
+function getTimestamp() {
```

Comment:
Unfortunately, test setup is run after jQuery is sourced so this would freeze `getTimestamp` to use the initial, non-mocked `window.performance`.